app/models/post.rb

```diff
@@ -1,2 +1,13 @@
 class Post < ActiveRecord::Base
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+  after_save :update_posts_counter
```

Comment:
- [ ] OPTIONAL
Good job for invoking `after_save` method on `update_posts_counter` method, however kindly mark `update_posts_counter` as private after the invoke, similar to :
```ruby
  # app/models/post.rb
  after_save :update_posts_counter

  private

  def update_posts_counter
    author.increment!(:posts_counter)
  end